languageFactories/wdl-biscayne/src/main/scala/languages/wdl/biscayne/WdlBiscayneLanguageFactory.scala

```diff
@@ -38,18 +38,34 @@ class WdlBiscayneLanguageFactory(override val config: Config) extends LanguageFa
 
     val checked: Checked[ValidatedWomNamespace] = for {
       _ <- enabledCheck
-      bundle <- getWomBundle(workflowSource, source.workflowOptions.asPrettyJson, importResolvers, factories)
-      executable <- createExecutable(bundle, source.inputsJson, ioFunctions)
+      bundleWithImports <- getWomBundleWithImports(workflowSource, source.workflowOptions.asPrettyJson, importResolvers, factories)
+      executable <- createExecutable(bundleWithImports._1, source.inputsJson, ioFunctions)
```

Comment:
TOL: Could this be a `case class` to avoid having to use the `_1` syntax?

Reply:
This is no longer needed. Please see [this](https://github.com/broadinstitute/cromwell/pull/5073#discussion_r303998619) discussion.